### PR TITLE
Export of PDF gene panel report using PDFKit

### LINKED ISSUE #3125: Find all the report exported to PDF using Weasyprint and refactor to use PDFKit
as in #3122 and #3120

- [x] case delivery report (cases viewa) ---> #3127
- [x] download_hpo_genes (cases views) ---> #3132
- [ ] panel_export (panels views) ---> #3133

So we can remove the Weasyprint dependency

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR:
- Library switch (PDFKit instead of Weasyprint) for downloading gene panels reports (fix #3125 )
- Fixes the loading spinner that keeps spinning after loading a gene panel report
- Loading of gene panel reports is much faster now

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. On cg-vm1, **main branch**: go to a gene panel and click on **Export to PDF**:
image
1. Make sure that the loading spinner keeps going after the download is finished
2. Switch to this branch and repeat, you should see that:
- [ ] download of the report is faster
- [ ] The spinner stops after the download

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Keep portrait orientation in PDF delivery report
 - Export delivery report to PDF using PDFKit instead of Weasyprint
 - PDF export of clinical and research HPO panels using PDFKit instead of Weasyprint
+- Export delivery report to PDF using PDFKit
```
Comment (dnil) [inline CHANGELOG.md:21]:
```suggestion
```
Duplicated gist from above!

File: scout/server/blueprints/panels/templates/panels/panel.html [hunk 2]
```diff
@@ -102,7 +102,7 @@
         </li>
       {% endif %}
       <li class="list-group-item">
-        <a href="{{ url_for('panels.panel_export', panel_id=panel._id) }}" class="btn btn-warning btn-sm">Export to PDF</a>
+        <a href="{{ url_for('panels.panel_export', panel_id=panel._id) }}" class="btn btn-warning btn-sm" download>Export to PDF</a>
```
Comment (dnil) [inline scout/server/blueprints/panels/templates/panels/panel.html:105]: 🆗

File: scout/server/blueprints/panels/views.py [hunk 3]
```diff
@@ -197,14 +202,21 @@ def panel_export(panel_id):
...
-    return render_pdf(
-        HTML(string=html_report),
-        download_filename=data["panel"]["panel_name"]
-        + "_"
-        + str(data["panel"]["version"])
-        + "_"
-        + datetime.datetime.now().strftime("%Y-%m-%d")
-        + "_scout.pdf",
+
+    bytes_file = html_to_pdf_file(html_report, "portrait", 300)
```
Comment (dnil) [inline scout/server/blueprints/panels/views.py:206]: 💯

**Review by dnil — APPROVED** [review]

**Comment by dnil** [comment]
:ok:
Screenshot 2022-02-04 at 09 41 09

**Comment by dnil** [comment]
Actually, it would probably be ok to remove WeasyPrint with that?

**Comment by northwestwitch** [comment]
> Actually, it would probably be ok to remove WeasyPrint with that?

I'm working on a new PR about it, there are some lib changes as well, that's why I say that error. I'm opening it as soon as it works!

**Comment by dnil** [comment]
Yeah, just saw the comment in https://github.com/Clinical-Genomics/scout/issues/3135. Good, I'll merge this in the meantime.

**Pull request merged**